qiskit-superstaq/qiskit_superstaq/superstaq_backend.py

```diff
@@ -120,6 +120,22 @@ def run(
 
         return job
 
+    def retrieve_job(self, job_id: str) -> qss.SuperstaqJob:
+        """Gets a job that has been created on the Superstaq API.
+
+        Args:
+            job_id: The UUID of the job. Jobs are assigned these numbers by the server during the
+            creation of the job.
+
+        Returns:
+            A `qss.Job` which can be queried for status or results.
```

Comment:
Should this be: 
```suggestion
            A `qss.SuperstaqJob` which can be queried for status or results.
```
?